lib/awful/mouse/init.lua

```diff
@@ -298,12 +298,42 @@ function mouse.object.set_current_client() end
 function mouse.object.get_current_wibox()
     local obj = capi.mouse.object_under_pointer()
     if type(obj) == "drawin" then
-        return obj.wibox()
+        return obj.wibox
     end
 end
 
 function mouse.object.set_current_wibox() end
 
+--- Get the widgets currently under the mouse cursor.
+--
+-- @property current_widgets
+-- @tparam widget|nil The widget
```

Comment:
ldoc has a funny syntax, "param" here is turned into "type". I could make a "@type" alias, but it would also be misleading.

But there really is an error, it should be `@tparam widget|nil widget The widget`